notebooks/Composer_Functional.ipynb

```diff
@@ -209,7 +213,7 @@
    "source": [
     "As we might expect, adding data augmentation doesn't help us when we aren't training long enough to start overfitting.\n",
     "\n",
-    "Now let's try using some algorithms that modify the model. We're going to keep things simple and just add a [Squeeze-and-Excitation](https://docs.mosaicml.com/en/latest/method_cards/squeeze_excite.html) module after the larger of the two conv2d operations in our model. We show examples of how to use other algorithms as well as a reference."
+    "Instead, let's try using some algorithms that modify the model. We're going to keep things simple and just add a [Squeeze-and-Excitation](https://docs.mosaicml.com/en/latest/method_cards/squeeze_excite.html) module after the larger of the two conv2d operations in our model."
```

Comment:
This isn't really "instead", since the data augmentation algorithms are still applied within the training dataloader.

Reply:
Done

Reply:
I moved the custom_method_tutorial changes to a separate branch, but I'm not sure how to adjust the rewrite that git did. I just made the changes in the notebook locally (and ran `make clean-notebooks`, so should be no outputs) and git created the diff the way it did. They weren't really huge changes.

I think for Colab, I will have to create a new notebook on Colab and then point the docs to them? I suggest we merge these changes to the local notebooks first since this PR has been pending for a while, and then I will update the colab versions.